python/.pre-commit-config.yaml

```diff
@@ -55,3 +55,4 @@ repos:
       - id: forbid-new-submodules
       - id: mixed-line-ending
       - id: trailing-whitespace
+        exclude: .bumpversion.cfg|.dockerignore
```

Comment:
This time I let bump2version update the .bumpversion.cfg file and added an exlusion to the trailing whitespace pre-commit hook so these two automated tools weren't fighting.